Source/Scene/TileBoundingRegion.js

```diff
@@ -99,8 +115,52 @@ define([
 
         var ellipsoid = defaultValue(options.ellipsoid, Ellipsoid.WGS84);
         computeBox(this, options.rectangle, ellipsoid);
+
+        /**
+         * An oriented bounding box that encloses this tile's region.
+         * This is used to calculate tile visibility.
+         *
+         * @type {OrientedBoundingBox}
+         * @default OrientedBoundingBox()
+         *
+         * @private
+         */
+        this._orientedBoundingBox = OrientedBoundingBox.fromPoints(getBoxPoints(this.rectangle, this.minimumHeight, this.maximumHeight));
```

Comment:
I believe we can use `OrientedBoundingBox.fromRectangle` here.  For example, see https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Workers/createVerticesFromQuantizedTerrainMesh.js#L121